resolwe_bio/processes/workflows/amplicon.yml

```diff
@@ -199,15 +206,26 @@
         input:
           bam: '{{steps.bwa_trim}}'
 
-- slug: workflow-accel-2
+- slug: workflow-accel-gatk4
   category: workflow:primary
-  name: Accel Amplicon Pipeline
+  name: Accel Amplicon Pipeline (GATK4)
   data_name: "Pipeline ({{ reads|sample_name|default('?') }})"
   requirements:
     expression-engine: jinja
   version: 0.0.2
```

Comment:
Raise version.

Reply:
This is a workflow with a new slug, so the version should be low. I will change this to `0.0.1`